Configuration.cs

```diff
@@ -10,6 +10,9 @@ public sealed class Configuration : IPluginConfiguration
 
     #region Saved configuration values
 
+    public TitleGroup randomTitleGroup { get; set; }
+    public bool       assignRandomTitleOnAreaChange { get; set; }
```

Comment:
Properties should be capitalized.

```suggestion
    public bool AssignRandomTitleOnAreaChange { get; set; }
```